raft/src/main/java/org/apache/kafka/raft/LeaderState.java

```diff
@@ -319,6 +328,10 @@ public String name() {
     }
 
     @Override
-    public void close() {}
+    public void close() {
+        if (accumulator != null) {
```

Comment:
When would accumulator be null?

Reply:
QuorumState tests where I was passing in null for the accumulator to get the code to compile. I'll see about using a mock.